apps/twap/src/app/plugins/infura.ts

```diff
@@ -0,0 +1,41 @@
+import { SupportedChainId } from '@cowprotocol/cow-sdk';
+import { ethers } from 'ethers';
+import { FastifyInstance } from 'fastify';
+import { ComposableCoW__factory } from '@cow-web-services/abis';
+import fp from 'fastify-plugin';
+
+const NETWORKS = ['goerli'];
+export const COMPOSABLE_COW_ADDRESS: Record<SupportedChainId, string> = {
+  1: '0xF487887DA5a4b4e3eC114FDAd97dc0F785d72738',
+  100: '0xF487887DA5a4b4e3eC114FDAd97dc0F785d72738',
+  5: '0xF487887DA5a4b4e3eC114FDAd97dc0F785d72738',
+};
+
+const SINCE = 9289470;
```

Comment:
Why this block specifically?
If there is a reason, we could add some comment as description.